repl/src/test/scala/org/apache/spark/repl/ReplSuite.scala

```diff
@@ -34,33 +33,33 @@ class ReplSuite extends SparkFunSuite {
   def runInterpreter(master: String, input: String): String = {
     val CONF_EXECUTOR_CLASSPATH = "spark.executor.extraClassPath"
 
-    val in = new BufferedReader(new StringReader(input + "\n"))
-    val out = new StringWriter()
+
     val cl = getClass.getClassLoader
-    var paths = new ArrayBuffer[String]
-    if (cl.isInstanceOf[URLClassLoader]) {
-      val urlLoader = cl.asInstanceOf[URLClassLoader]
-      for (url <- urlLoader.getURLs) {
-        if (url.getProtocol == "file") {
-          paths += url.getFile
-        }
-      }
-    }
-    val classpath = paths.map(new File(_).getAbsolutePath).mkString(File.pathSeparator)
 
     val oldExecutorClasspath = System.getProperty(CONF_EXECUTOR_CLASSPATH)
-    System.setProperty(CONF_EXECUTOR_CLASSPATH, classpath)
+    if (oldExecutorClasspath == null) {
+      System.clearProperty(CONF_EXECUTOR_CLASSPATH)
+    } else {
+      System.setProperty(CONF_EXECUTOR_CLASSPATH, oldExecutorClasspath)
+    }
```

Comment:
I don't understand your changes to the handling of CONF_EXECUTOR_CLASSPATH.  The old code saved the previous value, modified it, ran the tests, then restored the original value (which probably should have been in a `finally`).  But it seems you're just resetting it to the same value it already has?

Reply:
This is my error, let me fix that. That behavior should be retained.